src/ui/layout/tutorial.md

```diff
@@ -244,219 +393,287 @@ class MyApp extends StatelessWidget {
       ],
     );
   }
-}
 ```
 
-The function adds the icon directly to the column.
-The text is inside a `Container` with a top-only margin,
-separating the text from the icon.
+### Position the buttons with a `Row` widget
 
-Build the row containing these columns by calling the
-function and passing the color, `Icon`, and text specific
-to that column. Align the columns along the main axis
-using `MainAxisAlignment.spaceEvenly` to arrange the
-free space evenly before, between, and after each column.
-Add the following code just below the
-`titleSection` declaration inside the `build()` method:
+Add the following code into the `ButtonSection` widget.
 
-<?code-excerpt "lib/main.dart (buttonSection)" title?>
+1. Call the helper method three times, once for each button.
```

Comment:
Same here, it's not a helper method, it's a widget. Also be specific what one (`BuildButtonColumn`).

Reply:
Fixed.